packages/theme/src/cli/services/package.ts

```diff
@@ -0,0 +1,53 @@
+import {zip} from '@shopify/cli-kit/node/archiver'
+import {fileExists, readFile} from '@shopify/cli-kit/node/fs'
+import {AbortError} from '@shopify/cli-kit/node/error'
+import {renderSuccess} from '@shopify/cli-kit/node/ui'
+import {resolvePath} from '@shopify/cli-kit/node/path'
+
+const themeDirectoriesPattern = [
+  'assets',
+  'config',
+  'layout',
+  'locales',
+  'sections',
+  'snippets',
+  'templates',
+  'release-notes.md',
+].join('/**|')
+
+export async function packageTheme(inputDirectory: string) {
```

Comment:
I know, we don't have a real convention, but in e.g. `open.ts` or `publish.ts`, exported function has the same name as the file.
Therefore, was there a reason why `packageTheme` was not simply called `package`?
Import path `packages/theme/src/cli/services/package.ts` should contain enough context to make clear that it's about _themes_ :)

Reply:
![image](https://user-images.githubusercontent.com/18431672/214323777-c6c3d27c-54d0-438b-80f2-c778a2eaa1bf.png)

package is a reserved word, i can add that in a comment for context. i figured it was better to only change the function name and not the file name so it matches the rest of the commands. this makes it easy to find files

Reply:
This is of course a valid argument to keep `packageTheme` and `package.ts` :)